### ISSUE TITLE
What is your result tested on wider face?

### ISSUE THREAD

**Issue by Edwardmark** [louis-she/sfd.pytorch#28]

**Comment by louis-she** [comment]
Havn't tested it yet. My own mAP on all wider face validation set is 0.7 mAP as said in README.md.
One of the trainer reached 0.78mAP on hard and 0.83mAP on easy using this code.
